mixer/adapter/svcctrl/template/svcctrlreport/template.proto

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package svcctrlReport;
+
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
+import "mixer/v1/template/extensions.proto";
+import "mixer/v1/config/descriptor/value_type.proto";
+
+option (istio.mixer.v1.template.template_variety) = TEMPLATE_VARIETY_REPORT;
+
+message Template {
```

Comment:
Can we add comments to this Template, with example configuration and description of expected values?

Reply:
done. I have sent out a separated PR just for template. We can first go through it then this PR.